src/phoenix/datasets/evaluators/base.py

```diff
@@ -270,11 +267,26 @@ def _validate_sig(fn: Callable[..., Any], fn_name: str) -> None:
         raise ValueError(f"`{fn_name}` should allow variadic keyword arguments `**kwargs`")
 
 
+class CodeEvaluator(Evaluator, ABC, is_abstract=True):
+    """
+    A convenience super class for defining code evaluators.
+
+    This class is intended to be subclassed, and should not be instantiated directly.
+    """
+
+    _kind = AnnotatorKind.LLM
```

Comment:
Isn't this the same as LLM evaluator? Isn't evaluator by default CODE evaluators?